bridge/client/app/app.module.ts

```diff
@@ -307,5 +301,6 @@ const ktbModules = [
     },
   ],
   bootstrap: [KtbRootComponent],
+  exports: [],
```

Comment:
This `exports` was probably introduced automatically by the IDE, but is not necessary.